keep/api/models/db/migrations/versions/2024-10-08-22-23_0796be640663.py

```diff
@@ -0,0 +1,54 @@
+"""Add  runbook tables
```

Comment:
I would appreciate if both migrations will be merged together.

Reply:
ack